ruby-challenges.rb

```diff
@@ -10,7 +10,28 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+# -----Pseudo Code----
+# INPUT: number
+# OUTPUT: string stating whether or not the number is odd or even
+# create a function call odd_or_even that takes in a number
+# create a conditional state that if the modulo 2 of the number == zero then we return the its even
+# else we return that its false
 
+def odd_or_even num 
+    if num % 2 == 0
```

Comment:
Nicely done! Look at even? if you feel like using a few other tools Ruby provides